lib/thread.c

```diff
@@ -40,6 +41,12 @@ DEFINE_MTYPE_STATIC(LIB, THREAD_STATS,  "Thread stats")
 #include <mach/mach_time.h>
 #endif
 
+#define AWAKEN(m) \
```

Comment:
It is used in multiple places, but looks like it snuck out of funcname_thread_add_read_write in a force push somewhere...and yes, but the pipe poke related changes are actually quite small, just the bits in fd_poll and thread_master_create